runner_benchmark/taskset.py

```diff
@@ -14,6 +14,9 @@
 
 
 class SurveyRunnerTaskSet(TaskSet, QuestionnaireMixins):
+    wait_time = between(int(os.getenv('USER_WAIT_TIME_MIN_SECONDS', 1)),
+                        int(os.getenv('USER_WAIT_TIME_MAX_SECONDS', 2)))
```

Comment:
So this waits between the task executions not requests therefore we need to reintroduce the manual wait time in `replay_requests` method.